CSGOSimple/CSGOSimple.vcxproj

```diff
@@ -14,7 +14,7 @@
     <ProjectGuid>{F3E42845-8D56-4BB3-821D-8163AB1337F0}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>CSGOSimple</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
```

Comment:
Is there any reason why any adjustments to this file are necessary?

Reply:
no